lib/rgb_matrix/engine.ex

```diff
@@ -41,15 +41,17 @@ defmodule RGBMatrix.Engine do
     GenServer.cast(__MODULE__, {:set_animation, animation_type})
   end
 
+  @typep frame :: %{LED.t() => RGBMatrix.any_color_model()}
```

Comment:
:warning: (recommended)

This may be better as a public type since it's used in the return value of a public function.